lib/main.js

```diff
@@ -9,6 +9,7 @@ const mime = require('mime-types');
 const { Transform } = require('readable-stream');
 const params = require('./params');
 const MetaStorage = require('./meta-storage');
+const Metrics = require('@podium/metrics');
```

Comment:
Is this published publicly? I don't think it is...

It's not really coupled to podium, so might might want to open source it under `@finn-no/metric-stream` or something like that?

Reply:
Correct currently only works on my machine due to the private nature of the metrics module.